### ISSUE TITLE
Hotkey to minimise to system tray

### ISSUE THREAD

**Issue by Harjot1Singh** [getferdi/ferdi#857]
**Is your feature request related to a problem? Please describe.**
I'm an ex-Franz user. Love Ferdi! The main thing I miss being able to `ctrl-w` to minimise to the system tray. 

There is no explicit menu entry to do this, and so it relies on users knowing that they can click "x" or the system tray icon

I'd love to have a way of doing this again without the mouse.

**Describe the solution you'd like**
Add a menu item + hotkey to minimise Ferdi (`ctrl-w`). Franz puts these under the `Window` submenu, it seems sensible to add it there (and potentially remove the "open todos drawer", which is a duplicate of the "todos" menu).

As an added bonus, it'd be lovely to have a global hotkey to pull up Ferdi again, but I do appreciate that this would be a little more involved.

**Describe alternatives you've considered**
- Using my mouse and sucking it up: can live with it but not ideal.
- Switching back to Franz itself:  na
- Windows Key + Down: Actually has the same effect (intended?), but I don't think it's obvious, and don't know what the equivalent is on mac.

**Comment by avielc** [comment]
@vantezzen - Hi mate,
Let me just add, it used to work on previous version (there are old issues open here which have reference to it on Mac, it also exists on the Windows client.
I noticed that during the start up process of Fredi, you have a couple of seconds where Ctrl+W Does minimize the app, 
But after that, both the option from the "File menu" and the shortcut stops working.

tl;dr - definitely a bug mate. Would really appreciate if it could be fixed, otherwise i'll need to see if an older version is good enough to be used (Compared to Franz\Rambox - this shortcut is built in, in both apps)

**Comment by fuzzy76** [comment]
This irritates me daily. Command-W is the universal "close tab or window" shortcut on macOS, and it's a pain to have to grab for the mouse every time I'm done IM'ing...

**Comment by stnkl** [comment]
@vraravam Looks like this has already been implemented and the issue can be closed.

**Comment by vraravam** [comment]
Thanks @stnkl - I knew `Cmd+M` worked for all apps on macos - but, I wasn't aware whether it worked in other OSes. Thanks for confirming. We can close this ticket now!

**Comment by vraravam** [comment]
@all-contributors - please add @stnkl for bug
